compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -31,10 +31,10 @@ class CompilationTests extends ParallelTesting {
 
   // Positive tests ------------------------------------------------------------
 
-  // @Test  // enable to test compileStdLib separately with detailed stats
+  @Test  // enable to test compileStdLib separately with detailed stats
   def compileStdLibOnly: Unit = {
     implicit val testGroup: TestGroup = TestGroup("compileStdLibOnly")
-    compileList("compileStdLib", TestSources.stdLibSources, scala2Mode.and("-migration", "-Yno-inline", "-Ydetailed-stats"))
```

Comment:
stdlib is already tested with the same options in `pos`. We should only run the test once, since it is expensive. I don't care whether we keep it as part of `pos` or make it a separate test.